docs/DeveloperGuide.md

```diff
@@ -287,6 +287,101 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 2.
 
+**Use case: Add a tag**
+
+**MSS**
+
+1. User requests to add a new tag.
+2. StonksBook add the provided tag.
+
```

Comment:
Think you're missing a `    Use case ends.` here and in some other portions below as well